ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -28,6 +28,11 @@ flag asserts
   manual: False
   default: False
 
+flag expensive-invariants
+  description: Enable checks for expensive invariants
+  manual: False
```

Comment:
I think we want the flag to manual, to avoid the solver turning it on automatically somehow (though I don't think it would happen currently as it doesn't control any dependencies).